src/WebJobs.Script/Config/FunctionsHostingConfiguration.cs

```diff
@@ -77,10 +77,6 @@ public string GetValue(string key, string defaultValue = null)
                             _configs = configs;
                             _logger.LogDebug($"Updaiting FunctionsHostingConfigurations '{settings}'");
                         }
-                        else
-                        {
-                            throw new InvalidOperationException("FunctionsHostingConfigurations does not exists");
-                        }
```

Comment:
?

Reply:
We do not want to throw exception if the file does not exists.